web/cobrands/sass/_layout.scss

```diff
@@ -614,6 +615,34 @@ body.authpage {
   bottom: 2em;
 }
 
+.big-hide-pins-link {
+  display: block;
+  position: absolute;
+  top: 248px; // top of #fms_pan_zoom_zoomout + height of #fms_pan_zoom_zoomout + 22px gap
+  right: 38px; // right of #fms_pan_zoom (0.5em) + right of #fms_pan_zoom_panup (30px)
```

Comment:
I guess you could put this inside fms_pan_zoom to combat having to do this, but might not be worth it.

Reply:
Don’t think it’s worth it. Also, it has nothing to do with panning or zooming :-(